web_recipients_hide/__init__.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
```

Comment:
use short headers as in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py